packages/pn-personagiuridica-webapp/src/components/Notifications/GroupSelector.tsx

```diff
@@ -33,7 +35,11 @@ const GroupSelector: React.FC<Props> = ({ currentGroup, onGroupSelection }) => {
   }, []);
 
   return (
-    <Box data-testid="groupSelector" display="inline-block" sx={{ verticalAlign: 'middle', mx: 2 }}>
+    <Box
+      data-testid="groupSelector"
+      display="inline-block"
```

Comment:
change this property with `display={{ xs: 'block', lg: 'inline-block' }}`

Reply:
done